data_generator.py

```diff
@@ -51,20 +51,22 @@ def transform_image(self, img, bbox):
     return new_img, cropped_width, cropped_height, new_bbox[0], new_bbox[1]
   
   def transform_bbox(self,bbox):
-    x,y,w,h = [round(i) for i in bbox] # (x,y,w,h) anchored to top left
-    center_x = int(x+w/2)
-    center_y = int(y+h/2)
-    new_w = w if w >= h else round(h * self.input_dim[0]/self.input_dim[1])
-    new_h = h if w <  h else round(w * self.input_dim[1]/self.input_dim[0])
-    new_x = int(center_x - new_w/2)
-    new_y = int(center_y - new_h/2)
-    return (new_x,new_y,new_x+new_w,new_y+new_h)
+    x,y,w,h = [i for i in bbox] # (x,y,w,h) anchored to top left
+    center_x = x+w/2
+    center_y = y+h/2
+    new_w = w if w >= h else h * self.input_dim[0]/self.input_dim[1]
```

Comment:
kind of a nit: I feel it would be easier to read if you did 
```
if w >= h:
    stuff
else:
    other stuff
```

Reply:
Hands are typing Python, but my heart is typing Ruby ;) haha for sure I will switch.